src/main/daml/Daml/Finance/Lifecycle/Effect.daml

```diff
@@ -26,12 +26,21 @@ template Effect
       -- ^ The effect's description.
     targetInstrument : InstrumentKey
       -- ^ The target instrument.
+      --   For example, in the case of a swap instrument, this would be the original instrument
+      --   version before lifecycling, that contains the current swap payment.
     producedInstrument : Optional InstrumentKey
-      -- ^ The produced instrument, when it exists.
```

Comment:
Maybe this belongs to the effect interface rather than the implementation?

Reply:
I added the comments to both now.